probe/docker/container.go

```diff
@@ -212,6 +213,7 @@ func (c *container) GetNodeMetadata() report.NodeMetadata {
 		ContainerCreated: c.container.Created.Format(time.RFC822),
 		ContainerCommand: c.container.Path + " " + strings.Join(c.container.Args, " "),
 		ImageID:          c.container.Image,
+		ContainerIPs:     c.container.NetworkSettings.IPAddress,
```

Comment:
Two comments:
1. Don't you want to include the `SecondaryIPAddresses` too?
2. I think you should provide a mechanism to easily parse the addresses in case there's more than one. For the docker labels I am encoding them in json.